<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>gonext_103393.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>gonext_103393.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>720</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Bold</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(12, 15, 154);background-color: qlineargradient(spread:pad, x1:1, y1:0.704, x2:1, y2:0, stop:0 rgba(18, 46, 148, 255), stop:1 rgba(66, 71, 110, 255));font: 75 14pt &quot;KoPub돋움체 Bold&quot;;color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>검사하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>720</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pngwing.com.png</normaloff>pngwing.com.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>430</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>140</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>470</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>gonext_103393.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>gonext_103393.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serial_line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>750</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(74, 74, 74);</string>
   </property>
   <property name="text">
    <string>시리얼 번호 입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>magnifying-glass.png</normaloff>magnifying-glass.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="user_label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="serial_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>75</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>1041</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(253, 253, 253);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <pointsize>15</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 178, 178);
font: 57 15pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>결과</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_1</zorder>
  <zorder>label_5</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>btn_1</zorder>
  <zorder>reset_btn</zorder>
  <zorder>label_6</zorder>
  <zorder>label_3</zorder>
  <zorder>label_10</zorder>
  <zorder>label_8</zorder>
  <zorder>comboBox</zorder>
  <zorder>serial_line</zorder>
  <zorder>label</zorder>
  <zorder>search_btn</zorder>
  <zorder>user_label</zorder>
  <zorder>serial_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
